Musicplayer.php

```diff
@@ -13,28 +13,37 @@
 	</head>
 <body>
 	<div class ="grid-container">
-		<div class="img_1">
```

Comment:
21/9: Delete the banner because in this layout I decided to make the nav bar to be at the top because I released that the nav bar was importation then the banner So I will delete it for now and apply it latter.